assets/js/translator.js

```diff
@@ -635,16 +635,14 @@ function populateTranslationList() {
 
     $('#srcLangSelect').val(curSrcLang);
     var currentSrcLangSelect = $('#srcLangSelect option[value="' + $('#srcLangSelect').val() + '"]');
-    // Check if language begins with &nbsp;
-    if(currentSrcLangSelect.text().charAt(0) === ' ') {
-        // If it does, temporarily remove the four spaces
-        currentSrcLangSelect.text(currentSrcLangSelect.text().substring(4)); // eslint-disable-line no-magic-numbers
+    if(currentSrcLangSelect.text().startsWith(' ')) {
```

Comment:
There's no need to do this with one whitespace here or below. Just use the four spaces.